### DISCUSSION TITLE
Is there a 64-bit version of Neutone?

### DISCUSSION THREAD

**Discussion by Jon104** [QosmoInc/neutone_sdk#38] [Q&A] [answered]
I've been trying to use Neutone with Cubase Pro 12 without success.
Cubase does not seem to support 32-bit plugins.

I was wondering if there was a 64-bit version of Neutone?
If not, has anyone already tried to build it in 64 bits (M1 chips) ?
If so, what were the conclusions? 

Thank you so much ! 😍

**Comment by christhetree** [comment, marked as answer]
Hi Jon, thanks for asking! 

On OSX this is actually an issue with Cubase and not with Neutone because the plugin is 64 bit and is detected as such by all other DAWs. However, Cubase 11 and 12 detect it as 32 bit for some reason when running on rosetta. 

Try downloading Cubase 12 and make sure it's the Apple silicon version, not the rosetta version, and Neutone should be detected correctly as 64 bit. Let us know how it goes!

**Reply by Jon104** [reply]
Thank you for your response.

It did work!

Rosetta is used by default when running Cubase on the Mac Apple silicon. 
BUT, when you go to Finder -> Cubase -> Get Info -> General -> [ ] Open using Rosetta.
Open using Rosetta is unchecked (even if it is actually running by default).

So i had to check Open using Rosetta -> Close -> Reopen Get Info -> Uncheck Open using Rosetta -> TADA!

**Reply by christhetree** [reply]
Glad you got it working!
